src/olympia/users/models.py

```diff
@@ -241,9 +241,10 @@ def picture_url(self):
             ])
             return user_media_url('userpics') + path
 
-    @amo.cached_property
+    @amo.cached_property(writable=True)
```

Comment:
In the tests, it's needed to delete it... that parameter is a particularity of our own implementation (I should really revisit #3396 at some point), django's doesn't have it and happily lets you write/delete cached properties by default.